modules/checkbox/react/lib/Checkbox.tsx

```diff
@@ -57,7 +58,7 @@ const CheckboxInputWrapper = styled('div')<Pick<CheckboxProps, 'disabled'>>(
       height: checkboxHeight,
       transition: 'box-shadow 150ms ease-out',
       width: checkboxWidth,
-      zIndex: 1,
+      pointerEvents: 'none',
```

Comment:
What is this for?

Reply:
When I added `display: 'flex',`, it created an issue where the checkbox wouldn't register clicks and this seemed to fix the issue but I removed it and it still work 🤷‍♂. So I'm going to remove it.